src/js/modal-dialog.js

```diff
@@ -454,17 +461,25 @@ class ModalDialog extends Component {
    * @listens keydown
    */
   handleKeyDown(event) {
+    const spatialNavigation = this.player_.spatialNavigation;
+    const isSpatialNavlistening = spatialNavigation && spatialNavigation.isListening && !spatialNavigation.isPaused;
 
-    // Do not allow keydowns to reach out of the modal dialog.
-    event.stopPropagation();
+    // Do not allow keydowns to reach out of the modal dialog unless spatialNavigation is enabled.
+    if (!isSpatialNavlistening) {
+      event.stopPropagation();
+    }
 
-    if (keycode.isEventKey(event, 'Escape') && this.closeable()) {
+    // If 'Esc' is pressed or Backspace is pressed & spatialNavigation is enabled & Modal is 'closeable'.
+    if (keycode.isEventKey(event, 'Escape') || (keycode.isEventKey(event, 'Backspace') && isSpatialNavlistening) && this.closeable()) {
       event.preventDefault();
       this.close();
       return;
+    // If 'Enter' is pressed & spatialNavigation is enabled & handleClick is available.
+    } else if (keycode.isEventKey(event, 'Enter') && isSpatialNavlistening && this.handleClick) {
```

Comment:
Similar question as the above, but for handleClick

Reply:
> Similar question as the above, but for handleClick

In this case we only want to handle click if the spatial-navigation is enabled.